Docker-Swarm-deployment/single-node/resource-server/rs-stack.yaml

```diff
@@ -7,21 +7,19 @@ networks:
 
 services:
   rs:
-    image: ghcr.io/datakaveri/rs-depl:3.0-03915b1
+    image: iudx/rs-depl:latest
     networks:
       - overlay-net
     secrets:
       - source: config
         target: /usr/share/app/secrets/configs/config.json
-      - source: keystore
-        target: /usr/share/app/secrets/configs/keystore.jks
     env_file:
       - ./.rs.env
     read_only: true
     volumes:
-      - type: tmpfs
```

Comment:
please keep this volume also. this is where the vertx stores the cache! this is applicable for all vertx servers,